lib/Config.ts

```diff
@@ -150,4 +150,36 @@ class Config {
   }
 }
 
+export const convertConfigToToml = () => {
```

Comment:
I think it would be better to have this in our `tasks` folder, and add it to gulp.
It will also make the npm script more straightforward, and not relying on a previous compilation.

Reply:
Will check that tomorrow too, thanks for the comment